client/app/visualizations/chart/chart-editor.html

```diff
@@ -32,6 +32,9 @@
       </div>
     </div>
 
+    <div ng-if="['line', 'area', 'column'].indexOf(options.globalSeriesType) >= 0" class="checkbox">
```

Comment:
Why scatter plot doesn't support annotations?

Reply:
You're right, it supports. Looks a bit strange, but will add - maybe, someone will need it